Core/OfficeDevPnP.Core/Utilities/WebParts/IWebPartPostProcessor.cs

```diff
@@ -5,6 +5,6 @@ namespace OfficeDevPnP.Core.Utilities.WebParts
 {
     public interface IWebPartPostProcessor
     {
-        void Process(WebPartDefinition wpDefinition, File webPartPage);
+        void Process(WebPartDefinition wpDefinition, Web web);
```

Comment:
You're right... that's my bad, I had to make all this stuff internal on initial implementation.    

From the other side, there is absolutely no point for the user to implement this interface, because all this stuff is used internally (`IWebPartPostProcessor` used from `private static WebPartDefinition AddWebPart`). In that case can we leave everything as is?